cmd/airflow_test.go

```diff
@@ -191,21 +198,16 @@ func (s *AirflowSuite) cleanUpInitFiles() {
 		"airflow_settings.yaml",
 		"packages.txt",
 		"requirements.txt",
-		"dags/exampledag.py",
-		"plugins/example-plugin.py",
 		"include",
 		"plugins",
 		"README.md",
-		".astro/config.yaml",
-		".astro/test_dag_integrity.py",
+		".astro",
 		"./astro",
-		"tests/dags/test_dag_example.py",
-		"tests/dags",
 		"tests",
 		"dags",
 	}
 	for _, f := range files {
-		e := os.Remove(f)
+		e := os.RemoveAll(f)
```

Comment:
Is there some way we can clean this up in one call? Seems very brittle to list these out individually.

Reply:
maybe create and run the tests in a temp dir, and delete the temp dir itself at the end of each test as part of `TearDownSubTest`

Reply:
Updated all tests to make use of temp directory (which will then be removed after tests are executed) instead of `cmd` directory